virtual_rainforest/models/abiotic_simple/simple_regression.py

```diff
@@ -0,0 +1,677 @@
+            .isel(time=time_index)
+            .expand_dims(
+                dim={"layers": np.arange(len(layer_roles) - layer_roles.count("soil"))},
+                axis=0,
+            ),  # vertical projection
+            DataArray(
+                np.full((layer_roles.count("soil"), int(data.grid.cell_area)), np.nan),
+                dims=["layers", "cell_id"],
+                coords={
+                    "layers": np.arange(
+                        len(layer_roles) - layer_roles.count("soil"), len(layer_roles)
+                    ),
+                },
+            ),
+        ],
+        dim="layers",
+    )
+    atmospheric_pressure = atmospheric_pressure_1.assign_coords(
+        {
+            "layers": np.arange(0, len(layer_roles)),
+            "layer_roles": ("layers", layer_roles[0 : len(layer_roles)]),
+            "cell_id": data.grid.cell_id,
+        },
+    )
+    output.append(atmospheric_pressure)
+
+    # Mean atmospheric C02 profile, [ppm]
+    atmospheric_co2_1 = xr.concat(
+        [
+            data["atmospheric_co2_ref"]
+            .isel(time=time_index)
+            .expand_dims(
+                dim={"layers": np.arange(len(layer_roles) - layer_roles.count("soil"))},
+                axis=0,
+            ),  # vertical projection
+            DataArray(
+                np.full((layer_roles.count("soil"), int(data.grid.cell_area)), np.nan),
+                dims=["layers", "cell_id"],
+                coords={
+                    "layers": np.arange(
+                        len(layer_roles) - layer_roles.count("soil"), len(layer_roles)
+                    ),
+                },
+            ),
+        ],
+        dim="layers",
+    )
+    atmospheric_co2 = atmospheric_co2_1.assign_coords(
+        {
+            "layers": np.arange(0, len(layer_roles)),
+            "layer_roles": ("layers", layer_roles[0 : len(layer_roles)]),
+            "cell_id": data.grid.cell_id,
+        },
+    )
+    output.append(atmospheric_co2)
+
+    # Precipitation at the surface is reduced as a function of leaf area index
+    precipitation_surface = data["precipitation"].isel(time=time_index) * (
+        1 - water_interception_factor * data["leaf_area_index"].sum(dim="layers")
+    )
+
+    # Mean soil moisture profile, [] and Runoff, [mm]
+    soil_moisture, surface_run_off = calculate_soil_moisture(
+        layer_roles=layer_roles,
+        precipitation_surface=precipitation_surface,
+        current_soil_moisture=data["soil_moisture"],
+        soil_moisture_capacity=soil_moisture_capacity,
+    )
+
+    output.append(soil_moisture)
+    output.append(surface_run_off)
+    return output
+
+
+# supporting functions
+def logarithmic(x: DataArray, gradient: float, intercept: float) -> DataArray:
+    """Logarithmic function.
+
+    Args:
+        x: x values
+        gradient: gradient
+        intercept: intercept
+
+    Returns:
+        y values
+    """
+    return DataArray(gradient * np.log(x) + intercept)
```

Comment:
I think that creating a `DataArray` again is not needed and you can simply do:
```suggestion
    return gradient * np.log(x) + intercept
```

Reply:
I get a mypy error here if I remove the DataArray because I set the output type to DataArray.